### ISSUE TITLE
Only show outer edge of Radar Coverage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#124]
```
Game version: 0.93.450


Operating System: Windows 7 64-bit


Java runtime version: 1.7.0_01


Installed using the Launcher? Yes


Game language (en, hu): EN


This is an enhancement, based on the current Issue list I believe this is the 
place for these.

This is also a minor enhancement in my eyes, just aesthetically pleasing to 
see, no actual functional changes.

My suggestions is, would it be possible to have the radar fields of your 
planets in particular to only show the outer edge.

As it just looks a little messy as you expand and the radar fields expand, you 
get loads of overlapping edges - http://screencast.com/t/a2lEmaP5i
Here is a slightly better example from the original game - where things tend to 
get a bit messy - http://screencast.com/t/t1KxA4VI - 
http://screencast.com/t/t1KxA4VI

Thanks for reading.


```

Original issue reported on code.google.com by `liam0...@gmail.com` on 2 Nov 2011 at 9:50

**Comment by GoogleCodeExporter** [comment]
```
Hi and thanks for reporting. Imperium Galactica 2 had the union-type radar 
display you are suggesting. I need to think about the non-trivial algorithm.
```

Original comment by `akarn...@gmail.com` on 2 Nov 2011 at 10:55
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**, **Priority-Low**, **Milestone-0.98**
- Removed labels: **Type-Defect**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Available in SVN.
```

Original comment by `akarn...@gmail.com` on 3 Nov 2011 at 3:29
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fix: 0.93.475
```

Original comment by `akarn...@gmail.com` on 15 Dec 2011 at 10:39
- Changed state: **Fixed**
